src/server/ApiRoutes/ApiRoutes.js

```diff
@@ -32,7 +32,10 @@ router
 
 router
   .route('/hold/request/:bibId-:itemId')
-  .get(Hold.newHoldRequestServer)
+  .get(Hold.newHoldRequestServer);
```

Comment:
What? No. This is `get`ting the page. There's a `post` below that handles the no-js scenario when requesting a hold. The `itemSource` should not be in the url, but it's okay when posting it to the API.